assets/scss/03-organisms/_stacked-row-section.scss

```diff
@@ -1,70 +1,57 @@
-.ma__stacked-row-section {
+.ma__stacked-row {
+  position: relative;
 
-  @media ($bp-large-max) {
-    padding-bottom: 45px;
+  &__section {
 
     & ~ & {
-      border-top-width: 1px;
-      border-top-style: solid;
+      border-color: $c-bd-divider;
+
+      @media ($bp-large-max) {
+        border-top-width: 1px;
+        border-top-style: solid;
+      }
+
+      .ma__stacked-row__title .ma__comp-heading {
+        margin-top: 15px;
+      }
     }
 
     & ~ &--borderless {
-      border-top-width: 0;
+
+      @media ($bp-large-max) {
+        border-top-width: 0;
+      }
     }
+
   }
 
   &__container {
-    @include ma-container();
+    @include ma-container;
   }
 
   // reduce the width of the title and border
-  &--restricted + &--restricted &__container {
+  &__section--restricted + &__section--restricted &__container {
     @include ma-container('restricted');
   }
 
-
-  & ~ & &__container:before {
-    display: block;
+  &__section ~ &__section &__container:before {
+    border-color: $c-bd-divider;
     content: "";
-    padding-top: 45px;
+    display: block;
 
     @media ($bp-large-min) {
       border-top-width: 1px;
       border-top-style: solid;
-      padding-top: 90px;
     }
   }
 
-  & ~ &--borderless &__container:before {
+  &__section ~ &__section--borderless &__container:before {
     @media ($bp-large-min) {
       border-top-width: 0;
     }
   }
 
-  &--restricted &__title {
+  &__section--restricted &__title {
     max-width: 820px;
   }
-
-  .main-content {
-
-    @media ($bp-large-max) {
-      margin-bottom: 0;
-    }
-
-    .page-content > * {
-      @include ma-component-spacing;
-    }
-  }
-}
-
-//theme

```

Comment:
Theme specific styles are separated from the common styles, so everybody knows where to touch for theme related changes.

Reply:
Based on my conversations with @clair0917 combining the theme sections was desirable. If that is not the case I can put these back. It just seemed odd because these were often declaring the same thing that was already set above.